xAPI.md

```diff
@@ -255,6 +255,25 @@ learning that SCORM could not enable.
 
 ## 1.0 Summary of the Experience API
 
+The Experience API (Application Programming Interface) - or xAPI as it is commonly called - is a framework that 
```

Comment:
I prefer ```xAPI for short``` instead of ```or xAPI as it is commonly called```

I prefer ```specification``` to ```framework``` as xAPI is commonly referred to as a specification.